pkg/scraper/client.go

```diff
@@ -31,13 +31,14 @@ import (
 // KubeletInterface knows how to fetch metrics from the Kubelet
 type KubeletInterface interface {
 	// GetSummary fetches summary metrics from the given Kubelet
-	GetSummary(ctx context.Context, host string) (*stats.Summary, error)
+	GetSummary(ctx context.Context, info NodeInfo) (*stats.Summary, error)
 }
 
 type kubeletClient struct {
-	port            int
-	deprecatedNoTLS bool
-	client          *http.Client
+	port                     int
```

Comment:
Usage of `port` field now depends on kubeletClient. We should rename it to more explicit. I would propose to rename it to `defaultPort`